week-3/week-3-practice/london.js

```diff
@@ -0,0 +1,23 @@
+/*
+I am new to london and would like to know what transport I can take to different Locations.
+Using .map() print where I can go if I only want to use the river boat.
+
+An array with London Location has been provided
+Console.log has been provided
+*/
+
+var londonLocation = [
+  ["London Bridge", "tube", "bus", "river boat"],
+  ["Tower bridge", "tube", "bus"],
+  ["Greenwich", "tube", "bus", "river boat"]
+];
+
+var londonLocationTransport = londonLocation.map(function(item) {
```

Comment:
This works better and I tried to replace it but did not work for some reason 🤔
`var londonLocationTransport = [];`
`londonLocation.forEach(function(item) {`
  `if (item.length === 4) {`
   ` londonLocationTransport.push([item[0], item[3]]);`
 ` }`
`});`